### PR TITLE
Update 1546632293.rst

### PR DESCRIPTION
**Pull request by joworeiter**
Add Info that the getter has to also nullable (of course)

### CODE REVIEW

File: Documentation/Exceptions/1546632293.rst [hunk 1]
```diff
@@ -35,3 +35,25 @@ Add explicitly a default value:
...
    
 `Read more information on this behaviour <https://blog.reelworx.at/detail/typo3-extbase-and-type-annotations-with-php-74/>`__.
+
+TYPO3 11.5.31 - 2023-09-29
+==========================
+
+Situation
+---------
+
+When validating a controller's argument, the following exception is thrown::
```
Comment (brotkrueml) [inline Documentation/Exceptions/1546632293.rst:45]:
```suggestion
When validating a controller's argument, the following exception is thrown:
```

File: Documentation/Exceptions/1546632293.rst [hunk 2]
```diff
@@ -35,3 +35,25 @@ Add explicitly a default value:
...
+
+   #1546632293 RuntimeException Could not get value of property "Site\Site\Domain\Model\Booking::startdate",
```
Comment (brotkrueml) [inline Documentation/Exceptions/1546632293.rst:47]:
```suggestion
   .. code-block:: text
      #1546632293 RuntimeException Could not get value of property "Site\Site\Domain\Model\Booking::startdate",
```

File: Documentation/Exceptions/1546632293.rst [hunk 3]
```diff
@@ -35,3 +35,25 @@ Add explicitly a default value:
...
+   make sure the property is either public or has a getter getStartdate(), a hasser hasStartdate() or an isser isStartdate().
```
Comment (brotkrueml) [inline Documentation/Exceptions/1546632293.rst:48]:
```suggestion
      make sure the property is either public or has a getter getStartdate(), a hasser hasStartdate() or an isser isStartdate().
```

File: Documentation/Exceptions/1546632293.rst [hunk 4]
```diff
@@ -35,3 +35,25 @@ Add explicitly a default value:
...
+
+But there is a :php:`public function getStartdate(): DateTime` defined in the class.
+
+Solution
+--------
+
+In Getter, the return value has to be nullable:
```
Comment (brotkrueml) [inline Documentation/Exceptions/1546632293.rst:55]:
```suggestion
In the getter, the return value has to be nullable:
```

File: Documentation/Exceptions/1546632293.rst [hunk 5]
```diff
@@ -35,3 +35,25 @@ Add explicitly a default value:
...
+
+::
+
+  publich function getStartdate(): ?DateTime
```
Comment (brotkrueml) [inline Documentation/Exceptions/1546632293.rst:59]:
```suggestion
  public function getStartdate(): ?DateTime
```

File: Documentation/Exceptions/1546632293.rst [hunk 6]
```diff
@@ -35,3 +35,26 @@ Add explicitly a default value:
...
+
+TYPO3 11.5.31 - 2023-09-29
+==========================
+
+Situation
+---------
+
+When validating a controller's argument, the following exception is thrown:
+
+   .. code-block:: text
```
Comment (brotkrueml) [inline Documentation/Exceptions/1546632293.rst:47]:
```suggestion
.. code-block:: text
```

File: Documentation/Exceptions/1546632293.rst [hunk 7]
```diff
@@ -35,3 +35,26 @@ Add explicitly a default value:
...
+TYPO3 11.5.31 - 2023-09-29
+==========================
+
+Situation
+---------
+
+When validating a controller's argument, the following exception is thrown:
+
+.. code-block:: text
+      #1546632293 RuntimeException Could not get value of property "Site\Site\Domain\Model\Booking::startdate",
```
Comment (brotkrueml) [inline Documentation/Exceptions/1546632293.rst:48]:
```suggestion
   #1546632293 RuntimeException Could not get value of property "Site\Site\Domain\Model\Booking::startdate",
```

File: Documentation/Exceptions/1546632293.rst [hunk 8]
```diff
@@ -35,3 +35,26 @@ Add explicitly a default value:
...
+==========================
+
+Situation
+---------
+
+When validating a controller's argument, the following exception is thrown:
+
+.. code-block:: text
+   #1546632293 RuntimeException Could not get value of property "Site\Site\Domain\Model\Booking::startdate",
+      make sure the property is either public or has a getter getStartdate(), a hasser hasStartdate() or an isser isStartdate().
```
Comment (brotkrueml) [inline Documentation/Exceptions/1546632293.rst:49]:
```suggestion
   make sure the property is either public or has a getter getStartdate(), a hasser hasStartdate() or an isser isStartdate().
```

**Review by brotkrueml — APPROVED** [review]

**Comment by brotkrueml** [comment]
@joworeiter Thanks for your contribution :+1:

**Comment by joworeiter** [comment]
> @joworeiter Thanks for your contribution 👍

sry for the typos ;) you are welcome!

**Pull request merged**